### PR TITLE
Documentation fixups & additions

### PR DESCRIPTION
**Pull request by UmanShahzad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,89 +9,102 @@ Linux Kernel eBPF Collectors
 
 The respository has the following directory structure:
 
-- `artifacts`: Directory that will have the eBPF programs when the compilation
+- `artifacts`: directory that will have the eBPF programs when the compilation
   process ends.
-- `includes`: Common headers
-- `kernel`: The eBPF programs source code
+- `co-re`: contains all eBPF programs that utilize eBPF CO-RE (Compile Once -
+  Run Everywhere) technology.
```
Comment (thiagoftsm) [inline README.md:15]: Inside `co-re` directory we also have some testers, please, add them to this list.

File: README.md [hunk 2]
```diff
@@ -9,89 +9,102 @@ Linux Kernel eBPF Collectors
...
+- `docs`: contains an assortment of documentation related to this repository.
+- `includes`: headers used throughout the project.
+- `kernel`: contains all eBPF programs that don't utilize eBPF CO-RE
+  technology; these may be considered legacy more and more as time progresses
+  and we shift fully to CO-RE.
+- `kernel-patches`: contains patches needed to properly compile our legacy
+  `kernel/` eBPF programs on some kernel versions.
+- `libbpf`: this is a submodule'd fork of the official `libbpf` package, the
```
Comment (thiagoftsm) [inline README.md:23]: The submodule in this repo is a fork of https://github.com/netdata/libbpf, while `netdata/libbpf` is a fork of official libbpf, probably this additional details need to be explicit.

File: README.md [hunk 3]
```diff
@@ -103,29 +116,30 @@ when running the build image.
...
+## Manual Compilation
 
-After the necessary changes have been done inside the `kernel/Makefille` file,
-you need to run the following command to compile the eBPF programs:
+After you've got your `kernel/Makefille` properly setup, you can run the
+following command to compile all the eBPF programs:
 
 ```bash
-$ make
+$ make -j`nproc`
```
Comment (thiagoftsm) [inline README.md:131]: Please, add an additional information to explicit what means `nproc`.

Comment (UmanShahzad) [reply README.md:134]: Done in https://github.com/netdata/kernel-collector/pull/274/commits/ed80620d8818ca3e312d70b29f7b22f531587800

File: README.md (hunk 2 above)
Comment (UmanShahzad) [reply README.md]: Done in https://github.com/netdata/kernel-collector/pull/274/commits/ed80620d8818ca3e312d70b29f7b22f531587800

File: README.md (hunk 1 above)
Comment (UmanShahzad) [reply README.md:15]: Done in https://github.com/netdata/kernel-collector/pull/274/commits/ed80620d8818ca3e312d70b29f7b22f531587800

**Review by thiagoftsm — DISMISSED** [review]
Thank you very much @UmanShahzad , nice job! :)

File: README.md [hunk 4]
```diff
@@ -141,33 +158,34 @@ To release a new version and create a Github Release:
...
+   Netdata Installer.
 
 2. Wait for the CD pipeline to finish in the Github Actions UI.
 3. Review the Release, Updates Release Notes, etc in the Github Releases UI.
 4. Push the "Publish Release" button in the Github Releases UI.
 
 ## Contribution
 
-Netdata is an open-software software and we are always open for contributions that
+Netdata is open-software software and we are always open for contributions that
```
Comment (vlvkobal) [inline README.md:180]:
```suggestion
Netdata is open-source software and we are always open for contributions that
```

**Review by vlvkobal — APPROVED** [review]

**Review by erdem2000 — DISMISSED** [review]
@UmanShahzad thanks for your effort!

File: README.md (hunk 4 above)
Comment (thiagoftsm) [reply README.md:180]: I will adjust this in this #272 .

**Pull request merged**